k8s/prometheus-operator/README.md

```diff
@@ -129,6 +134,25 @@ Configure the container image:
 export IMAGE_OPERATOR="marketplace.gcr.io/google/prometheus-operator"
 ```
 
+#### If using GKE on-prem
```

Comment:
Could you add one sentence here to give reader more context:

"This solution can be used in [GKE On-Prem](https://cloud.google.com/anthos/gke/docs/on-prem) clusters."

Reply:
Done